src/skeleplex/measurements/fit_surface.py

```diff
@@ -0,0 +1,307 @@
+import networkx as nx  # noqa: D100
+import numpy as np
+import trimesh
+import trimesh as tri
+from scipy.interpolate import Rbf
+from scipy.spatial import Delaunay
+from sklearn.decomposition import PCA
+
+from skeleplex.graph.constants import (
+    EDGE_SPLINE_KEY,
+    LOBE_NAME_KEY,
+    NODE_COORDINATE_KEY,
+)
+
+
+def extract_central_region(points, percentile=50):
+    """
+    Extracts the central region of the point cloud using PCA and a distance threshold.
+
+    Parameters
+    ----------
+    points : np.ndarray
+        (n_points, 3) array containing the 3D point cloud.
+    percentile : float
+        The percentage of points to keep (default 50% around the mean).
+
+    Returns
+    -------
+    central_points : np.ndarray
+        The extracted central points.
+    pca : PCA
+        The PCA transformation object.
+    mean : np.ndarray
+        Mean of the original point cloud.
+    components : np.ndarray
+        The principal component axes.
+    """
+    # Center the points
+    mean = np.mean(points, axis=0)
+    centered_points = points - mean
+
+    # Apply PCA
+    pca = PCA(n_components=3)
+    pca.fit(centered_points)
+    components = pca.components_  # Principal axes
+
+    # Transform points into PCA space
+    aligned_points = centered_points @ components.T  # Manual projection
+
+    # Compute distances in PCA space (only in the XY plane)
+    distances = np.linalg.norm(aligned_points[:, :2], axis=1)
+    threshold = np.percentile(distances, percentile)
+
+    # Select only the central points
+    central_points = points[distances < threshold]
+    return central_points, mean, components
+
+
+def fit_surface_to_pointcloud_rbf_pca(points: np.ndarray, smooth=0.2, percentile=100):
+    """Fits an RBF surface to a pointcloud using PCA for rotation invariance.
```

Comment:
Can you please define write out Radial Basis Function (assuming this is what RBF is).

Also, is there a ref you can link to at the bottom of the docstring?